css/css-borders/tentative/border-radius-side-shorthands/border-radius-side-shorthands-001.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Borders and Box Decorations 4 Test: Physical 'border-*-radius' shorthands set the two radii of one side.</title>
+<link rel="author" title="Sebastian Zartner" href="mailto:sebastianzartner@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-borders-4/#corner-sizing">
+<link rel="match" href="../../reference/border-radius-side-shorthands-ref.html">
+<meta name="assert"
+  content="This test checks that the physical 'border-*-radius' shorthands set the corresponding 'border-[*]-radius' longhands.">
+
+<style>
+div {
+  width: 100px;
+  height: 100px;
+  margin: 10px;
+}
+
+#refenence-shape-1,
+#refenence-shape-2,
+#refenence-shape-3,
+#refenence-shape-4 {
+  border: 10px solid red;
```

Comment:
I'm a little concerned about the test assertion here.  With antialiased edges, the idea that drawing green thing on top of identically-shaped red thing will cover all of the red isn't really true, since antialiasing is done on the assumption that the coverage areas are independent.

Maybe the answer is that once these features are implemented, browser developers will just add an appropriate `meta name=fuzzy` annotation to all of these tests.

But probably a better answer is -- at least if it's possible -- to make the *reference* do the exact same drawing of a green thing on top of an identically shaped red thing, so that the reference has the same visual artifacts that the test has, and thus the test and reference should really match.

Reply:
I agree with you on the antialiasing issue. But I believe this can be worked out or adjusted later. Sebastian's tests do not absolutely need to use the overlapping technique for his tests... and if his tests need to, I am sure we can find a work-around for him.

Reply:
I just got to know about `meta name=fuzzy` the other day. I might that, though I'd very much need to guess the values. Therefore, it might be better to adjust the tests once implementors start to work on that.